user/src/test/java/com/jumar/user/utils/DateUtilsTest.java

```diff
@@ -0,0 +1,31 @@
+package com.jumar.user.utils;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.sql.Timestamp;
+import java.time.LocalDate;
+import java.sql.Date;
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DateUtilsTest {
+    private String dateOfBirth;
+
+    @Test
+    void converted_dateOfBirth_matches_correct_LocalDateTime_format() {
```

Comment:
this is testing the LocalDateTime class more than anything else - not business logic. We want our tests to test business logic.